### ISSUE TITLE
"schema_version"

### ISSUE THREAD

**Issue by stuartmcalpine** [LSSTDESC/dataregistry#77]
There is a mixture of references in the code to `schema` or `schema_version` in the code to mean the same thing.

Can we standardize this to `schema`? Particularly now that schema themselves now have versions, and this is just referring to their name.

**Comment by JoanneBogart** [comment]
Yes, I agree.  `schema_version` is old - and confusing - terminology.  Should just be `schema`.
